### PR TITLE
 #167878987 Connect sign in api endpoint to database

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?
Connects sign-in API endpoint to the database

#### Description of Task to be completed?
- Modified the verify email method in the models.
- Modified the user sign in controller methods.
- Modified the sign in validation. 

#### How should this be manually tested?
CLone this repository and run `npm install && run test`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 #167878987

#### Screenshots (if appropriate)
Screenshot from 2019-08-14 10-02-27

### CODE REVIEW

File: server/api/controllers/userController.js [hunk 1]
```diff
@@ -6,16 +7,22 @@ import User from '../models/user';
...
 
 		const passwordHashed = HashedPassword.hashPassword(password);
 		const newUser = User.createNewUser({
-			email, first_name, last_name, passwordHashed,
+			email,
+			first_name,
+			last_name,
+			passwordHashed,
 		});
 		// eslint-disable-next-line max-len
```
Comment (oesukam) [inline server/api/controllers/userController.js:16]: It isn't a best practice to every time disable eslint's rules in your code

**Review by oesukam — CHANGES_REQUESTED** [review]
@nziokaivy Your travis build is failing

File: server/api/controllers/userController.js (hunk 1 above)
Comment (nziokaivy) [reply server/api/controllers/userController.js:16]: I have fixed most of them now

**Comment by nziokaivy** [comment]
I have fixed it now the build is passing

**Review by oesukam — APPROVED** [review]

**Pull request merged**